sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -308,7 +310,15 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
 
           ret = rc.getResponseCode
           if (ret != 0) {
-            console.printError(rc.getErrorMessage())
+            // For analysis exception, only the error is printed out to the console.
+            rc.getException() match {
+              case e : AnalysisException =>
+                console.printError(
+                  s"""Failed with exception ${e.getClass.getName}: ${e.getMessage}""")
```

Comment:
I would just say, "Error in query: ${e.message}" or something.  It will always be `org.apache.spark.sql.AnalysisException` and we want to make it clear that is likely the users fault in this case (i.e. not a bug).